watson.zsh-completion

```diff
@@ -10,6 +10,7 @@ _watson_commands=(
   'help:Display help information'
   'log:Display each recorded session during the...'
   'merge:Perform a merge of the existing frames with a...'
+  'rename:Renames tag or project'
```

Comment:
Help message should be written in imperative form, i.e. "Rename tag..." instead of "Rename**s**"".